spanner/api/Spanner.Samples.Tests/QueryDataWithNumericParameterTest.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Xunit;
+
+[Collection(nameof(SpannerFixture))]
+public class QueryDataWithNumericParameterTest
```

Comment:
The main thing here is that we prefer to have one test per sample. Even if there's a little of code duplication or execution. With something like this, if the test failed, you don't know which part failed at first glance, sometimes the stack trace gets truncated on the test output, etc.

Reply:
Cool, that makes sense. I've now added separate tests. The only downside is that the tests are not asserting anything since the final test that everything works is really when you run the query data sample.

But I suppose at least we know that an exception wasn't thrown if each test completes.

Reply:
Yes, this is fine. If the one adding the column fails, probably all will fail, but at least we can quicly know which test is to blame.